README.rst

```diff
@@ -44,11 +48,26 @@ Features
 - support for several shared storage systems (Ceph)
 - support for several container technologies (Docker)
 
+Getting started
+---------------
+
+You can `install REANA locally <http://docs.reana.io/development/deploying-locally/>`_ , `deploy it at scale on premises
+<http://docs.reana.io/development/deploying-at-scale/>`_ (in about 10 minutes) or use https://reana.cern.ch. Once the system
+is ready, you can follow the guide to run `your first example <http://docs.reana.io/getting-started/first-example/>`.
+For more in depth information visit the `official REANA documentation <http://docs.reana.io/>`_.
+
+Community
+---------
+
+- Discuss [on Forum](https://forum.reana.io/)
+- Chat [on Gitter](https://gitter.im/reanahub/reana)
+- Follow us [on Twitter](https://twitter.com/reanahub)
```

Comment:
markdown -> rst

Now that I see this, what about using markdown for this README? It's the standard and since we don't need to include pieces of this in any readthedocs...